menu/cbs/menu_cbs_title.c

```diff
@@ -1142,6 +1157,10 @@ static int menu_cbs_init_bind_title_compare_label(menu_file_list_cbs_t *cbs,
          action_get_core_updater_list},
       {MENU_ENUM_LABEL_DEFERRED_CORE_MANAGER_LIST,
          action_get_core_manager_list},
+#ifdef HAVE_MIST
```

Comment:
This should be an either/or - don't bind `MENU_ENUM_LABEL_DEFERRED_CORE_MANAGER_LIST` to `action_get_core_manager_list` if `HAVE_MIST` is defined

Reply:
Oh wait - I misread. The following `MENU_ENUM_LABEL_DEFERRED_CORE_MANAGER_LIST` should be `MENU_ENUM_LABEL_DEFERRED_CORE_MANAGER_STEAM_LIST`, right...?